apps/condo/domains/contact/components/EditContactForm.tsx

```diff
@@ -37,13 +38,14 @@ export const EditContactForm: React.FC = () => {
     const ContactNotFoundMessage = intl.formatMessage({ id: 'Contact.NotFound.Message' })
     const ProfileUpdateTitle = intl.formatMessage({ id: 'EditingContact' })
     const NameLabel = intl.formatMessage({ id: 'field.FullName.short' })
-    const FullNamePlaceholderMessage = intl.formatMessage({ id:'field.FullName' })
+    const FullNamePlaceholderMessage = intl.formatMessage({ id: 'field.FullName' })
     const PhoneLabel = intl.formatMessage({ id: 'Phone' })
     const ExamplePhoneMessage = intl.formatMessage({ id: 'example.Phone' })
     const ExampleEmailMessage = intl.formatMessage({ id: 'example.Email' })
     const EmailLabel = intl.formatMessage({ id: 'field.EMail' })
     const ApplyChangesMessage = intl.formatMessage({ id: 'ApplyChanges' })
     const NoPermissionMessage = intl.formatMessage({ id: 'EditingContactNoPermission' })
+    const roleLabel = intl.formatMessage({ id: 'ContactRole' })
```

Comment:
```suggestion
    const RoleLabel = intl.formatMessage({ id: 'ContactRole' })
```

Reply:
Local variable. It's good to use camelCase.

Reply:
We already use CamalCase for translations in many place. Why you want to change this style?